src/docs/frontend/index.mdx

```diff
@@ -68,23 +68,28 @@ Both the `app/components/` and `app/views` folders contain React components.
 - Use a view for UI that will typically not be reused in other parts of the codebase
 - Use a component for UI that is designed to be highly reusable.
 
-### PropTypes
+### Type declarations
 
-Use them, be explicit, use the shared custom proptypes when possible.
+We do not follow any strict "prefer type over interface" rules for type declarations, so using either is at your discretion. That being said, there are [functional differences](https://www.typescriptlang.org/docs/handbook/2/everyday-types.html#differences-between-type-aliases-and-interfaces) between the two that you should be aware of so you can use them appropriately. 
```

Comment:
```suggestion
We do not follow any strict "prefer interface over type" rules for type declarations, so using either is at your discretion. That being said, there are [functional differences](https://www.typescriptlang.org/docs/handbook/2/everyday-types.html#differences-between-type-aliases-and-interfaces) between the two that you should be aware of so you can use them appropriately. 
```

Reply:
I would keep this as is now. Looking over the repo, it seems that prefer type was the unofficial guideline so I want to kind of debunk that with this line :)